.github/workflows/check.yml

```diff
@@ -50,9 +55,14 @@ jobs:
           sudo tlmgr update --self
           sudo tlmgr install latexmk isodate substr relsize ulem fixme rsfs extract layouts enumitem l3packages l3kernel imakeidx splitindex xstring
 
-      - name: make
+      - name: make (Linux)
+        if: matrix.cfg.os == 'ubuntu-22.04'
         run: make quiet
 
+      - name: make (MacOS)
+        if: matrix.cfg.os == 'macos-12'
+        run: make full
```

Comment:
So, this one is weird: the `||` in the Makefile seems to not work on MacOS anymore.

Reply:
Nothing is needed, but it's an odd change of behaviour, which I imagine should show up elsewhere... but I think we can live with it.

Reply:
The default shell changed from bash to zsh in modern macOS versions, but that shouldn't matter. It seems more likely that the behaviour of latexmk changed so that -silent causes it to exit with status 0 even if the build failed.

Reply:
Ohh, that's interesting! That means we'll see the same on Linux when that version changes, I expect!